Cython/Compiler/FusedNode.py

```diff
@@ -293,7 +293,7 @@ def _fused_instance_checks(self, normal_types, pyx_code, env):
 
     def _dtype_name(self, dtype):
         if dtype.is_typedef:
-            return '___pyx_%s' % dtype
+            return ('___pyx_%s' % dtype).replace(' ', '_')
```

Comment:
We do some form of additional underscore escaping in other places to avoid collisions resulting from something like `ctypedef const int const_int`. Seems worth doing here as well.

Also, seeing this here makes me think that we should use a dedicated prefix instead of plain `__pyx_`, to avoid even more risks of collisions with names generated elsewhere. Think of `ctypedef int v_var`, which would produce the local variable name `__pyx_v_var`.